packs/nest/actions/humidity.py

```diff
@@ -0,0 +1,14 @@
+from lib import actions
+
+class HumidityAction(actions.BaseAction):
+    def run(self, structure, device, target=None):
+        if target:
+            target_humidity = target
+        else:
+            target_humidity = self._get_device(structure, device).humidity
```

Comment:
Since it's explicit - if you are calling setter it's clear you are setting a value, likewise for the getter. You don't need to go check / dig into the code to see what's going on.

This approach is also not mutually exclusive with returning a value which was passed to the setter inside the setter function.